### ISSUE TITLE
AWS::Glue::Crawler does not accept !Ref to AWS::IAM::Role resource

### ISSUE THREAD

**Issue by phene** [aws-cloudformation/cfn-python-lint#883]
*cfn-lint version: 0.20.1*

`AWS::Glue::Crawler` spec: `E3008 Property "Role" has no valid Refs to Resources at Resources/Crawler/Properties/Role/Ref`

Example:
```yaml
Resources:
   ServiceRole:
    Type: AWS::IAM::Role
    Properties:
      RoleName: !Sub "${SystemName}-Glue-Service-${AWS::Region}"
      AssumeRolePolicyDocument:
        Version: "2012-10-17"
        Statement:
          - Effect: Allow
            Principal:
              Service:
                - glue.amazonaws.com
            Action:
              - "sts:AssumeRole"
  Crawler:
    Condition: IncludeCrawlResources
    Type: AWS::Glue::Crawler
    Properties:
      Name: !Sub "${SystemName}-warehouse"
      Description: Crawls data in the warehouse bucket
      DatabaseName: !Ref Database
      Schedule:
        ScheduleExpression: !Ref CrawlSchedule
      Role: !Ref ServiceRole
```
